package.json

```diff
@@ -108,7 +108,8 @@
     "karma-mocha-reporter": "^2.2.1",
     "karma-phantomjs-launcher": "^1.0.0",
     "karma-phantomjs-shim": "^1.4.0",
-    "karma-webpack-with-fast-source-maps": "^1.9.2",
```

Comment:
However, karma-webpack is very slow :(  I'll come back to this when I have some more time.